Put torchlike/signlike node on floor if paramtype2="none" (#11074)
minetest/minetest: doc/lua_api.txt

      used to compensate for how `glasslike` reduces visual thickness.
* `torchlike`
    * A single vertical texture.
    * If `paramtype2="[color]wallmounted":
        * If placed on top of a node, uses the first texture specified in `tiles`.
        * If placed against the underside of a node, uses the second texture
          specified in `tiles`.
        * If placed on the side of a node, uses the third texture specified in
          `tiles` and is perpendicular to that node.
    * If `paramtype2="none"`:

>         * It will be as if the node is always placed on the top of a node (see

Reviewer: ```suggestion
        * Will be rendered as if placed on the top of a node (see
```
